docs/Developers/Newcomers_Guide.rst

```diff
@@ -383,6 +383,25 @@ and clicking ``New Pull Request``.
 
 **Congratulations!** You have now created your first ``Pull Request``!
 
+Now, before having other contributors review your work, the system will do an
+automated code review and will comment on your Pull Request.
+You may be required to enhance your Pull Request by modifying your branch on
+your fork.
+
+.. important::
+
+    It is **never** necessary to close your Pull Request. **Never!**
+    It is **always** possible to modify your Pull Request using
+    `git <https://coala.io/git>`_.
+
+    You must not delete your fork, delete your branch, close your
+    ``Pull Request`` or start a new one etc. All of those actions will create
+    a mess.
```

Comment:
Instead of `start` this should be `open`. Also etc is not needed I think you have included all the scenarios.

Reply:
You're right.